staticcheck.conf

```diff
@@ -0,0 +1,15 @@
+initialisms = ["ACL", "API", "ASCII",
+	"CNI", "CPU", "CSS",
+	"DNS",
+	"EOF",
+	"GID", "GUID",
+	"HTML", "HTTP", "HTTPS",
+	"ID", "IP", "IPAM",
+	"JSON",
+	"QPS",
+	"RAM", "RPC",
+	"SLA", "SMTP", "SQL", "SSH",
+	"TCP", "TLS", "TTL",
+	"UDP", "UI", "UID", "URI", "URL", "UTF8", "UUID",
+	"VM",
+	"XML", "XMPP", "XSRF", "XSS"]
```

Comment:
See the comments on the submariner tree, I would add those at least too NAT / IP / IPSec